src/openfermion/ops/_interaction_rdm_test.py

```diff
@@ -13,9 +13,10 @@
 """Tests for interaction_rdms.py."""
 from __future__ import absolute_import
 
+import os
```

Comment:
`os` should be after `unittest` here I suspect

Reply:
They are both standard library modules so they should be placed in alphabetical order.